daemons/based/based_remote.c

```diff
@@ -508,6 +528,8 @@ cib_remote_msg(gpointer data)
         client->remote->auth_timeout = 0;
         client->name = crm_element_value_copy(command, PCMK_XA_NAME);
 
+        ncd->start_time = time(NULL);
```

Comment:
What is this for? Are we expecting another message?

Reply:
I don't remember what the point of this is anymore, so it's been cut.